### DISCUSSION TITLE
Nested instert [Apr 12]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#5512] [Discord Threads]
> 🙋‍♂️ **WuGGu**

**Comment by o1lab** [comment]
> 👨‍💻 **pranavxc**  
#### If you are using our normal api then you have to do multiple api calls... But if you are using our shared form view api then you can do it with a single api, in the backend handler we are taking care of nested mapping as well   
####    
#### Note: you can only map existing record and won't create if missing  
#  
> 👨‍💻 **pranavxc**  
#### What you have to pass is an array of objects with primary key and values  
#  
> 🙋‍♂️ **WuGGu**  
#### OK, thanks for the hint. I looked into the Chrome DevTools Logs and got it to work with, Postman. Is there any documentation about this? Because I didn't find one.  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1084530135094546472)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
